data/interfaces/default/settings.html

```diff
@@ -405,9 +405,9 @@
                         </div>
                         <div class="checkbox">
                             <label>
-                                <input type="checkbox" id="monitor_remote_access" name="monitor_remote_access" value="1" ${config['monitor_remote_access']}> Monitor Plex Remote Access
+                                <input type="checkbox" class="monitor-settings" id="monitor_remote_access" name="monitor_remote_access" value="1" ${config['monitor_remote_access']}> Monitor Plex Remote Access
```

Comment:
Actually it doesn't require either?

The scheduled task already runs, and the setting just toggles that if statement.